### DISCUSSION TITLE
Struggling to fine tune the pretrained CNN AI in the classification example

### DISCUSSION THREAD

**Discussion by Kiesmith** [bitcraze/discussions#547] [Q&A]
I have unzipped the christmas box dataset.

I am now at the /classification path trying to run: python3 train_classifier.py [--args]

When I run this, I get the error: "The Tensorflow library was compiled to use the AVX instructions, but these aren't available on your machine. Aborted (core dumped)."

I had to install TensorFlow prior to running this, I installed the latest. 

How do I fix this? 

**Another question**
I am doing this because I want to try running my own pre-trained AI (which I have stored on my main computer and I am expecting to somehow put it on the AI deck (not sure on the steps to do it yet). If I have a pretrained AI that I trained on my own, what would the steps be to program it? I am sure I will need to move it into the vm, put it somewhere and then upload it to the AI-deck. 

This will be really appreciated to solve. 
Thankyou,
Kieran

![image](https://user-images.githubusercontent.com/72498028/216419555-0b8cc0eb-4e29-49db-b8e4-92ea6b57f710.png)

**Comment by hmllr** [comment]
Hi :)
I guess you are following the classification demo documentation? Then you should install all requirements as specified here: https://github.com/bitcraze/aideck-gap8-examples/blob/master/examples/ai/classification/requirements.txt as described here: 
https://www.bitcraze.io/documentation/repository/aideck-gap8-examples/master/ai-examples/classification-demo/#tensor-flow
Hope this solves the problem,
Hanna

**Reply by Kiesmith** [reply]
I carried on the discussion here: https://github.com/bitcraze/discussions/discussions/559 
I went through the steps you provided but I had numerous issues. It shows the logs to what I did.
